### PR TITLE
Add /billing-csv to portal API docs

### PR DESCRIPTION
**Pull request by ollie-nye**
[Ticket: HEIM-264]

This documents the standard response from the portal API when requesting a billing csv

### CODE REVIEW

File: articles/portal/ptl-ref-portal-api.md [hunk 1]
```diff
@@ -3678,6 +3678,119 @@ Returns a CSV report for the whole month specified including the date provided.
...
+
+  Parameter name | Description | Type | Mandatory (Default)
+  ---------------|-------------|------|--------------------
+  period | The date for which you require the monthly report in the format YYYY-MM | String | Y
+  org_id | The ID of the compute service for which you want the billing report | String | Y
+
+#### Example request (Curl)
+
+```bash
+curl -b /tmp/cookies.txt -X GET -H "Accept: application/json" -k https://portal.skyscapecloud.com/api/billing/billing-csv?period=2018-01-&org_id=xx-xx-xx-xxxxxxx
```
Comment (m-digman) [inline articles/portal/ptl-ref-portal-api.md]: is the "-" correct after the period and before the "&org_id"?

Comment (ollie-nye) [reply articles/portal/ptl-ref-portal-api.md]: Removed in 98e1e97

File: articles/portal/ptl-ref-portal-api.md [hunk 2]
```diff
@@ -3678,6 +3678,119 @@ Returns a CSV report for the whole month specified including the date provided.
   Consumption (GB) | Average storage used over the month in gigabytes
   Cost | Amount that would be charged if this were for a whole month (in pounds)
 
+## GET /api/billing/billing-csv
+
+Returns a CSV containing billing data for the period given.
```
Comment (m-digman) [inline articles/portal/ptl-ref-portal-api.md]: Is it worth indicating this data only covers VMware, OpenStack and DR to the Cloud?

Comment (ollie-nye) [reply articles/portal/ptl-ref-portal-api.md]: Do any other services have org ids that could be used? If there are I'll add a note in

Comment (m-digman) [reply articles/portal/ptl-ref-portal-api.md]: Sorry don't know

**Pull request merged**